tests/sflow_test.py

```diff
@@ -261,6 +261,35 @@ def test_config_sflow_intf_sample_rate(self):
 
         return
 
+    def test_config_disable_all_intf(self):
```

Comment:
I dont see the test covering the app db entries or the 'show' command. ??

Reply:
I added the fault entries into appl_db.json and the `test_show_intf()` on line 55 will ensure we don't traceback due to the bad entries. 

The two new test I added `test_config_enable_all_intf` and `test_config_disable_all_intf` merely ensures the proper configDB entries get added when the user runs those commands.